Packs/CofenseIntelligenceV2/Integrations/CofenseIntelligenceV2/CofenseIntelligenceV2.py

```diff
@@ -172,45 +178,70 @@ def create_hr_for_cofense_search(threat: Dict):
     return threat_row
 
 
-def threats_analysis(severity_score: dict, threats: List, indicator: str, threshold: str):
+def extract_indicator_from_block(block: dict, command: str) -> str:
+    """Extract indicator from a block based on what command is being called.
+
+    Args:
+        block (dict): block from blockSet
+        command (str): the name of the command
+
+    Returns:
+        str: the value of the indicator
+    """
+    if command == "url":
+        try:
+            data = block.get("data_1", {}).get("url")
+        except AttributeError:
+            data = block.get("data_1")
+    else:
+        data = block.get("data_1")
+
+    return data
+
+
+def threats_analysis(severity_score: dict, threats: List, indicator: str, threshold: str, command: str):
     """ process raw response data and generate dbot score and human readable results
             Args:
                 - severity_score(dict): severity score mapping
                 - threats (list): threats data from cofense raw response
                 - indicator (string): threat severity level for dbot score calculation
                 - threshold (string): threshold for threat's severity
+                - command (string): name of the command
             return:
              Dict: represents human readable markdown table
              int: dbot score
     """
 
-    threshold_score = severity_score.get(threshold)
-    if threshold_score is None:
+    block_type: str = BLOCK_TYPE_MAPPING.get(command)
```

Comment:
```suggestion
    block_type: str = BLOCK_TYPE_MAPPING.get(command, "")
```
This should fix the current Mypy problem.